src/controllers/main.controller.spec.ts

```diff
@@ -0,0 +1,23 @@
+import "mocha";
+import * as sinon from "sinon";
+import { Response, Request } from "express";
+import { mainController } from "./main.controller";
+
+describe("Main Controller Tests", () => {
```

Comment:
Don't stub this test. Start a server, make an actual request to the API and assert the response